lms/templates/instructor/instructor_dashboard_2/cohort-form.underscore

```diff
@@ -38,7 +38,7 @@
                         <%- gettext('Students in this cohort are:') %>
```

Comment:
This isn't part of this PR, but I noticed it on the sandbox. This label is not following the conventions we have used for the other fields:

    Cohort Name
    Students in this cohort are:
    Associated Content Group

@catong do you have any suggestions?